### ISSUE TITLE
can we use this to compile tiff from multiple sources?

### ISSUE THREAD

**Issue by alexgman** [seikichi/tiff.js#46]
I have files like png, gif, jpeg, pdf, and I need to be able to compile all of these into 1 tiff file. Can we use this library to do that?

**Comment by johnthad** [comment]
No. If you look on the [Note](https://github.com/seikichi/tiff.js#note) on the README.md, you'll see that the create functions of LibTIFF are not supported. 

[photopea/UTIF.js](https://github.com/photopea/UTIF.js) says it [supports encoding TIFF](https://github.com/photopea/UTIF.js#encoding-tiff-images), but I've not tried it.

Using Java, I've merged various image types into PDF with [Apache PDFBox](https://pdfbox.apache.org/). If you _must have_ TIFF, you can try [jai-imageio/jai-imageio-core](https://github.com/jai-imageio/jai-imageio-core), but again this if Java, not JavaScript (and my only experience with it is saving 1-bit images as TIFF with CCITT T.6 compression).
